
diff --git a/build.gradle b/build.gradle
index e21609f..f082233 100644
--- a/build.gradle
+++ b/build.gradle
@@ -23,8 +23,8 @@ buildscript {
     jcenter()
     maven {
       url "https://plugins.gradle.org/m2/"
-    }
-  }
+    }
+   }
   apply from: file('gradle/buildscript.gradle'), to: buildscript
   apply from: "$rootDir/gradle/dependencies.gradle"

@@ -55,6 +55,13 @@ allprojects {

   repositories {
     mavenCentral()
+    maven {
+        url "http://repo1.acceldata.dev/repository/odp-central/"
+        credentials {
+           username = "maverick"
+           password = "df6f57652202571c0d3090808d38e660"
+             }
+       }
   }

   apply plugin: 'idea'
@@ -2061,6 +2068,11 @@ project(':connect:runtime') {
     testCompile project(':core').sourceSets.test.output

     testRuntime libs.slf4jlog4j
+    implementation 'org.mongodb.kafka:mongo-kafka:1.6.1.3.2.2.0-2-all:all@jar'
+    implementation 'com.google.cloud:google-cloud-shared-config:1.2.0.3.2.2.0-2@jar'
+    implementation 'io.aiven:aiven-kafka-connect:2.6.0.3.2.2.0-2@jar'
+    implementation 'io.aiven:aiven-kafka-connect-jdbc:6.1.2.3.2.2.0-2@jar'
+    implementation 'com.amazonaws:amazon-kinesis-kafka-connecter:0.0.8.3.2.2.0-2@jar'
   }

   javadoc {
